svm/tests/mock_bank.rs

```diff
@@ -89,6 +108,7 @@ fn load_program(name: String) -> Vec<u8> {
     dir.push(name.as_str());
     let name = name.replace('-', "_");
     dir.push(name + "_program.so");
+    std::println!("The dir is: {:?}", dir);
```

Comment:
Did you want this in here? Looks like maybe debugging?

Reply:
This was just for debugging. I missed it.